expansion/src/main/java/no/unit/nva/expansion/model/ExpandedResourceConversation.java

```diff
@@ -0,0 +1,64 @@
+package no.unit.nva.expansion.model;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import no.unit.nva.expansion.ResourceExpansionService;
+import no.unit.nva.expansion.WithOrganizationScope;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.service.impl.ResourceConversation;
+import no.unit.nva.publication.storage.model.Message;
+import nva.commons.apigateway.exceptions.NotFoundException;
+
+import java.net.URI;
+import java.util.Set;
+
+@JsonTypeName("ResourceConversation")
+public class ExpandedResourceConversation extends ResourceConversation implements WithOrganizationScope, ExpandedDataEntry {
+
+    private SortableIdentifier identifier;
```

Comment:
This is the publicationIdentifier. We should name it publicationIdentifier so that we will not think that the conversations get their own identifiers.